docs/content/guides/tools-and-building/custom-plugins.md

```diff
@@ -267,15 +271,50 @@ const hotInstance = new Handsontable(container, {
   [CustomPlugin.PLUGIN_KEY]: false,
 });
 ```
+:::
+
+::: only-for react
+```jsx
+import Handsontable from 'handsontable';
+import { CustomPlugin } from './customPlugin';
+
+<HotTable
+  // Pass `true` to enable the plugin with default options.
+  [CustomPlugin.PLUGIN_KEY]={true}
+  // You can also enable the plugin by passing an object with options.
+  [CustomPlugin.PLUGIN_KEY]={{
+    msg: 'user-defined message',
+  }}
+  // You can also initialize the plugin without enabling it at the beginning.
+  [CustomPlugin.PLUGIN_KEY]={false}
+/>
```

Comment:
AFAIK JSX does not support computed properties. I've changed the snippet to use the plugin key the same as the example above the section.
```suggestion
<HotTable
  // Pass `true` to enable the plugin with default options.
  customPlugin={true}
  // You can also enable the plugin by passing an object with options.
  customPlugin={{
    msg: 'user-defined message',
  }}
  // You can also initialize the plugin without enabling it at the beginning.
  customPlugin={false}
/>
```